RM_SUI/RM_SUI/View/Tabs/VPNInfoTab.swift

```diff
@@ -0,0 +1,120 @@
+//
+//  VPNTab.swift
+//  RM_SUI
+//
+//  Created by Дмитрий Супрун on 21.01.23.
+//
+
+import SwiftUI
+
+/// VPN connection Information
+struct VPNInfoTab: View {
+    // MARK: - Private Constants
+    private enum Constants {
+        static let ipAddressName = "IP Address"
+        static let ipValueName = "127.0.0.1"
+        static let receivedName = "Received"
+        static let receivedValueName = "0 Mb"
+        static let sentName = "Sent"
+        static let sentValueName = "0 Mb"
+        static let expireMessageName = "Service will expire after 15 days"
+        static let defaultFlagImageName = "ad"
+        static let flagImageShadowRadius: CGFloat = 15
+        static let vpnToggleFrameWidth: CGFloat = 200
+        static let flagImageStrokeWidth: CGFloat = 5
+        static let flagImageFrameWidth: CGFloat = 300
+        static let flagImageFrameHeight: CGFloat = 200
+        static let flagImageCornerRadius: CGFloat = 20
+        static let flagImageCornerRadiusDefault: CGFloat = 0
+        static let vpnInfoFrameWidth: CGFloat = 300
+        static let vpnToggleName = "VPN is active"
+        static let changeFlagImageName = "arrow.clockwise.circle"
+        static let verticalSpacing: CGFloat = 20
+        static let flagsList = ["ad", "ae", "af", "ag", "ai", "al", "am", "ao", "aq", "ar", "as", "at", "au", "aw", "ax"]
+    }
+    // MARK: - Public Properties
+    
+    var body: some View {
+        ZStack {
+            backgroundColor
+            VStack(spacing: Constants.verticalSpacing) {
+                Spacer()
+                countryFlagImageView
+                if vpnIsOn {
+                    changeCountryButton
+                }
+                vpnToggleView
+                    .frame(width: Constants.vpnToggleFrameWidth)
+                vpnInfo(name: Constants.ipAddressName, value: Constants.ipValueName)
+                vpnInfo(name: Constants.receivedName, value: Constants.receivedValueName)
+                vpnInfo(name: Constants.sentName, value: Constants.sentValueName)
```

Comment:
```suggestion
                vpnInfo(name: Constants.ipAddressName, value: Constants.ipValueName)
                vpnInfo(name: Constants.receivedName, value: Constants.receivedValueName)
                vpnInfo(name: Constants.sentName, value: Constants.sentValueName)           
```

по-хорощему конечно, все данные надо просить у вьюмодели, как это было в  плеере